src/pb_stub.h

```diff
@@ -152,6 +155,17 @@ class LogMessage {
 
 #define LOG_FL(FN, LN, LVL) LogMessage((char*)(FN), LN, LVL).stream()
 
+// The payload for the stub_to_parent message queue
+struct UtilsMessagePayload {
+  UtilsMessagePayload(
+      const PYTHONSTUB_CommandType& command_type, void* utils_message_ptr)
+      : command_type(command_type), utils_message_ptr(utils_message_ptr)
+  {
+  }
+  PYTHONSTUB_CommandType command_type;
+  void* utils_message_ptr;
+};
```

Comment:
If the buffer accepts `IPCMessage` class, we will need to reconstruct the message struct (e.g. [LogSendMessage](https://github.com/triton-inference-server/python_backend/blob/main/src/pb_stub.cc#L944) and [CleanupMessage](https://github.com/triton-inference-server/python_backend/blob/main/src/pb_stub.cc#L978C9-L979)) from `ipc_message->Args()` to [set the message handle](https://github.com/triton-inference-server/python_backend/blob/main/src/pb_stub.cc#L980-L981). The challenge is that since the buffer runs in a separate thread, we will need to make sure that all the objects used to form the `IPCMessage` before pushed to the buffer are still alive until the message is sent to the parent process. I'm not sure if modifying the `IPCMessage` class to handle the lifetime of those objects would be a good idea, hence created a new message payload.